src/AI/AIUtils.java

```diff
@@ -227,7 +232,7 @@ public static CommanderAbility queueCromulentAbility(Queue<GameAction> q, Comman
   {
     XYCoord origin = new XYCoord(unit.x, unit.y);
     Map<XYCoord, Double> shootableTiles = new HashMap<XYCoord, Double>();
-    ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap, false);
+    ArrayList<XYCoord> destinations = Utils.findPossibleDestinations(unit, gameMap, true);
```

Comment:
This is your function, but you changed the default value when you put the boolean back in.
Also it's a magic value blah blah.